src/mlpack/methods/emst/emst_main.cpp

```diff
@@ -34,8 +34,8 @@ PROGRAM_INFO("Fast Euclidean Minimum Spanning Tree", "This program can compute "
     "column corresponds to the distance between the two points.");
 
 PARAM_STRING_REQ("input_file", "Data input file.", "i");
-PARAM_STRING("output_file", "Data output file.  Stored as an edge list.", "o",
-    "emst_output.csv");
+PARAM_STRING_REQ("output_file", "Data output file.  Stored as an edge list.",
+    "o");
```

Comment:
We shouldn't require the user to specify and output file, but we should print a warning ("output will not be saved", I think mlpack_knn does this) if they don't specify -o.